resources/views/conferences/index.blade.php

```diff
@@ -41,43 +41,50 @@ class="glyphicon glyphicon-plus"
                 <ul class="list-conferences">
                     @forelse ($conferences as $conference)
                         <li>
-                            @if (Auth::check() && $conference->isFavorited())
-                                <a href="/conferences/{{ $conference->id  }}/unfavorite"
-                                   class="fav-button fav-button--faved"><span
-                                        class="glyphicon glyphicon-star"></a>
-                            @else
-                                <a href="/conferences/{{ $conference->id  }}/favorite"
-                                   class="fav-button"><span
-                                        class="glyphicon glyphicon-star-empty"></a>
-                            @endif
-                            <h3>
-                                <a href="{{ route('conferences.show', ['id' => $conference->id]) }}">{{ $conference->title }}</a>
-                            </h3>
-                            @if ($conference->cfpIsOpen())
-                                <span class="label label-info">CFP OPEN</span>
-                            @endif
-                            @if ($conference->joindin_id)
-                                <a href="http://joind.in/event/view/{{ $conference->joindin_id }}">
-                                    <span class="label-joindin"><img
-                                            src="/img/joindin-button.png"></span>
-                                </a>
-                            @endif
+                            <div class="conference-actions">
+                                @if (Auth::check() && !$conference->isFavorited())
+                                    @if ($conference->isDismissed())
+                                        <a href="/conferences/{{ $conference->id  }}/undismiss" class="action-button" title="I am interested in this conference"><span class="glyphicon glyphicon-plus"></a>
+                                    @else
+                                        <a href="/conferences/{{ $conference->id  }}/dismiss" class="action-button" title="I am not interested in this conference"><span class="glyphicon glyphicon-remove"></a>
+                                    @endif
+                                @endif
+                                @if (Auth::check() && !$conference->isDismissed())
```

Comment:
Our general rule in HTML is to include an extra linebreak between adjacent elements, including Blade control structures. So this would be:

```
...
@endif 

@if
... 
```

and elsewhere, there should be linebreaks between things like:

```
...
</div>

<div class="details">
...
```

Reply:
Got it, line breaks added.